pmm-app/src/pmm-qan-app-panel/module.ts

```diff
@@ -161,36 +168,50 @@ export class PanelCtrl extends MetricsPanelCtrl {
     const currentURL = new URL(url);
     const id = currentURL.searchParams.get('queryID') ? currentURL.searchParams.get('queryID') : '';
     const search = currentURL.searchParams.get('search') ? currentURL.searchParams.get('search') : '';
-    const filters = currentURL.searchParams.get('filters') ? currentURL.searchParams.get('filters') : this.retrieveFiltersFromVarParams(currentURL);
+    const filters = this.retrieveFiltersFromVarParams(currentURL);
     const mainMetric = currentURL.searchParams.get('main_metric') ? currentURL.searchParams.get('main_metric') : '';
     const columns = currentURL.searchParams.get('columns') ? currentURL.searchParams.get('columns') : '';
     const orderBy = currentURL.searchParams.get('order_by') ? currentURL.searchParams.get('order_by') : '';
     const groupBy = currentURL.searchParams.get('group_by') ? currentURL.searchParams.get('group_by') : '';
     const filterBy = currentURL.searchParams.get('filter_by') ? currentURL.searchParams.get('filter_by') : '';
     const activeDetailsTab = currentURL.searchParams.get('active_details_tab') ? currentURL.searchParams.get('active_details_tab') : '';
     const type = currentURL.searchParams.get('type') ? currentURL.searchParams.get('type') : '';
-
     // @ts-ignore
     return [id, type, search, filters, mainMetric, columns, orderBy, groupBy, filterBy, activeDetailsTab];
   }
 
   private retrieveFiltersFromVarParams(currentURL) {
-    const filtersFromVar = [];
-    PanelCtrl.retrieveVarParam(currentURL, filtersFromVar, 'var-host', 'node_name');
-    PanelCtrl.retrieveVarParam(currentURL, filtersFromVar, 'var-service', 'service_name');
-    PanelCtrl.retrieveVarParam(currentURL, filtersFromVar, 'var-db', 'database');
-    PanelCtrl.retrieveVarParam(currentURL, filtersFromVar, 'var-replication_set', 'replication_set');
-    PanelCtrl.retrieveVarParam(currentURL, filtersFromVar, 'var-environment', 'environment');
-    PanelCtrl.retrieveVarParam(currentURL, filtersFromVar, 'var-cluster', 'cluster');
-    return filtersFromVar.join(',');
+    const filtersList = [
+      'var-host',
+      'var-replication_set',
+      'var-environment',
+      'var-database',
+      'var-region',
+      'var-service',
+      'var-cluster',
+      'var-node_id',
+      'var-agent_id',
+      'var-service_id',
+      'var-az',
+      'var-node_type',
+      'var-node_model',
+      'var-node_name',
+      'var-service_name',
+      'var-service_type',
+      'var-username',
+    ];
```

Comment:
Could you please sort/group them in a similar way (for easier review and do not accidentally lose any of them):
https://github.com/percona/qan-api2/blob/master/services/analytics/base.go#L121